solutions/standard/catalogValidationValues.json.template

```diff
@@ -3,5 +3,5 @@
   "resource_group_name": $PREFIX,
   "tags": $TAGS,
   "existing_kms_instance_crn": $HPCS_US_SOUTH_CRN,
-  "kms_endpoint_url": "https://api.private.us-south.hs-crypto.cloud.ibm.com:8992"
+  "kms_endpoint_url": "https://e6dce284-e80f-46e1-a3c1-830f7adff7a9.api.private.us-south.hs-crypto.appdomain.cloud"
```

Comment:
oh, we should not have to do this. We should have a variable that replaces the value here (same way we have for `$HPCS_US_SOUTH_CRN`). I just created an issue to add that support to catalog pipeline, and will come back and change this when its ready

Reply:
Created https://github.ibm.com/GoldenEye/issues/issues/11341